app/views/partials/_nav-global.html.erb

```diff
@@ -0,0 +1,6 @@
+<% url_path = request.fullpath.split("/") %>
+<nav data-feature='navToggle' class="global-nav">
+    <a href="#">ground rules</a>
+    <a href="#" class="nav-item<% if url_path[0] == 'machines' %> selected <% end %>" id="machines">Machines</a>
+    <a href="#" class="nav-item<% if url_path[0] == 'roasts' %> selected <% end %>" id="roasts">Roasts</a>
```

Comment:
Maybe make those `href`(s) `href="#todo"` so you can find them and replace them easily later.

Or just talk to Jason and figure out what your URLs are going to be now and just finish this. I think you guys can agree on what the URLs should be very quickly and you can call this done without having to come back to it.